app/api/item/service.py

```diff
@@ -0,0 +1,8 @@
+from uuid import UUID
+from sqlalchemy.orm import Session
+
+from . import models
+
+
+def get_item_by_item_id(db: Session, item_id: UUID):
+    return db.query(models.Item).filter(models.Item.id == item_id).first()
```

Comment:
```suggestion
    return db.query(models.Item).filter(id == item_id).first()
```

如果是 select by primary key 的話只要寫 id 就好

https://docs.sqlalchemy.org/en/14/orm/session_basics.html#get-by-primary-key